File: sub-model/src/main/java/org/github/base/entity/SysUserPostEntity.java
package org.github.base.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import org.github.base.AbstractEntity;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableField;
import lombok.*;

/**
 * <p>
 * 用户与岗位关联表
 * </p>
 *
 * @author JYD_XL
 * @since 2019-11-21
 */
@Setter
@Getter
@EqualsAndHashCode(callSuper = true)
@TableName("sys_user_post")
public class SysUserPostEntity extends AbstractEntity {
    /** UID */
    private static final long serialVersionUID = 1L;

    /** 用户ID */
    @TableId("user_id")
    private Long userId;

    /** 岗位ID */
    @TableField("post_id")
    private Long postId;
}

File: sub-model/src/main/java/org/github/base/entity/SysUserPostEntity.kt
package org.github.base.entity

import com.baomidou.mybatisplus.annotation.TableName
import org.github.base.AbstractEntity

/**
 * <p>
 * 用户与岗位关联表
 * </p>
 *
 * @author JYD_XL
 * @since 2019-11-19
 */
@TableName("sys_user_post")
class SysUserPostEntity: AbstractEntity() {

    /** 用户ID */
    var userId: Long? = null

    /** 岗位ID */
    var postId: Long? = null

}
